### DISCUSSION TITLE
Let's make this great project even better

### DISCUSSION THREAD

**Discussion by viking1304** [foundation/foundation-emails#1141] [General]
I like this project very much, but I do not like the outdated information in the documentation. I also do not like missing support for new node versions.

**I am more than willing to help with both.**

I created a pull request https://github.com/foundation/foundation-emails-template/pull/70  that fixes an error when running the project with node v10.

I also created a pull request https://github.com/foundation/foundation-emails-template/pull/71 that adds support for node 16 and newer.

While we are waiting for the merge, you can use v12 or v14 with the [official repository](https://github.com/foundation/foundation-emails-template) 

```
❯ git clone https://github.com/viking1304/foundation-emails-template test
```

or you can temporarily use [my version](https://github.com/viking1304/foundation-emails-template) that works with all node versions from v10 to v19

```
❯ git clone https://github.com/viking1304/foundation-emails-template test
```

If you are using the official repo, be sure to switch to node v12 or v14 before installing modules.

```
❯ cd test
❯ nvm use v14
❯ npm install
```

If you use my repo, you can use any node you want (v10-v19)

```
❯ cd test
❯ npm install
```

To compile use:
```
❯ npm start
```

To compile everything and inline CSS:
```
❯ npm run build
```

I am also willing to help with documentation and potentially new examples.

I hope we can finally close off those issues after the merge:

https://github.com/foundation/foundation-emails/issues/1113 https://github.com/foundation/foundation-emails/issues/1120 https://github.com/foundation/foundation-emails/issues/1124 https://github.com/foundation/foundation-emails/issues/1127 https://github.com/foundation/foundation-emails/issues/1129 https://github.com/foundation/foundation-emails/issues/1139

**Comment by joeworkman** [comment]
Thank you. I will look at these and get this merged in. Much appreciated.

**Reply by viking1304** [reply]
May you please tell me if node v10 should be supported?

I want to make more PRs and update used libraries, but in some cases, we can't use the latest versions of those libraries if we need to maintain v10 compatibility.

If we want to maintain v10 compatibility, the foundation for emails framework itself should be patched too.

If you decide to ditch v10, I will close PR https://github.com/foundation/foundation-emails-template/pull/70

**Comment by elcaptain** [comment]
thanks so much for your work @viking1304 👏 
would be great to get your fixes merged!

**Comment by viking1304** [comment]
> thanks so much for your work @viking1304 👏 would be great to get your fixes merged!

I really hope @joeworkman will find some time to check and merge my fixes. There are some new PRs that can potentially clash with mine since they used the original version as a base to apply their fixes.

**Comment by simon-guider** [comment]
This works the first time. You saved me so much time. 

I hope your PR's get merged soon 

@joeworkman any news on this?
